db/data_migration/20230515143400_remove_central_african_republic_embassy_special_case.rb

```diff
@@ -0,0 +1,14 @@
+location = WorldLocation.find_by!(slug: "central-african-republic")
+organisation = WorldwideOrganisation.find_by!(slug: "british-embassy-kinshasa")
+organisation.world_locations += [location] unless organisation.world_locations.include?(location)
+
+location.reload
+
+embassy = Embassy.new(location)
+unless embassy.organisations_with_embassy_offices == [organisation]
```

Comment:
Is this all run in a transaction so raising will rollback? I wasn't sure it was failing fast to do this check at this point, but imagine it's because it's wrapped in a transaction

Reply:
Although there appears to be a precedent for [using `raise` in these data migrations](https://github.com/search?q=repo%3Aalphagov%2Fwhitehall+raise+path%3A%2F%5Edb%5C%2F%2F+path%3A%2F%5Edb%5C%2Fdata_migration%5C%2F%2F&type=code), you've made me realise that this probably isn't what I want to happen. I think it'll be good enough to write any unexpected data to stdout for us to manually investigate and fix. I'll make this change now.

Reply:
I've addressed this in https://github.com/alphagov/whitehall/pull/7658/commits/68706888d0165ca0de809c1916c3cb1c2571539e.